setup.py

```diff
@@ -33,13 +33,13 @@
     packages=['ftfy', 'ftfy.bad_codecs'],
     install_requires=['wcwidth'],
     tests_require=['pytest'],
-    python_requires='>=3.5',
+    python_requires='>=3.6',
     classifiers=[
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.5",
```

Comment:
Not really knowing how people use changelogs: is "no longer supports Python 3.5" something worth noting in the changelog?  Or will people get the information they need from `python_requires` and the lack of this line?